inst/templates/config.R

```diff
@@ -0,0 +1,77 @@
+# check the packages MSRV first
+source("tools/msrv.R")
+
+# check DEBUG and NOT_CRAN environment variables
+env_debug <- Sys.getenv("DEBUG")
+env_not_cran <- Sys.getenv("NOT_CRAN")
+
+# check if the vendored zip file exists
+vendor_exists <- file.exists("src/rust/vendor.tar.xz")
+
+is_not_cran <- env_not_cran != ""
+is_debug <- env_debug != ""
+
+if (is_debug) {
+  # if we have DEBUG then we set not cran to true
+  # CRAN is always release build
+  is_not_cran <- TRUE
+  message("Creating DEBUG build.")
+}
+
+if (!is_not_cran) {
+  message("Building for CRAN.")
+}
+
+# we set cran flags only if NOT_CRAN is empty and if
+# the vendored crates are present.
+.cran_flags <- ifelse(
```

Comment:
What's the point of dot-prefixed variables here?

Reply:
Just a visual indicator for myself to know that these are the variables that are going into the Makevars files. I can remove the prefix if it gives you the ick 😷

Reply:
Not really, just curious. `dot` prefixes I kinda in the 'wild west' of R, can't recall any styling rules regulating their usage.